infrastructure/code/template/details.html

```diff
@@ -46,18 +297,18 @@
 
     <!-- Data Flow -->
     <article class="data-flow-section">
-      <h2>${dataFlowHeading}</h2>
+      <h2 class="data-flow-header">${dataFlowHeading}</h2>
       <div class="data-flow-body">
         ${dataFlow}
       </div>
     </article>
 
     <!-- External example fixes -->
-    <article class="external-example-fixes">
-      <h2>External example fixes</h2>
-      <p id="info-top" class="font-light">
+    <article class="example-fixes">
```

Comment:
For the example fixes, how do we plan on swapping between the tabs? I think it makes more sense to add the JavaScript in here (`snyk-ls`) for changing what div is visible since it's mostly styling that will take care of that. Also, the behaviour of the tabs would stay the same regardless of IDE.

Reply:
We agreed I could do it in a follow-up PR as part of the ticket I'm working on